app/views/tickets/index.html.erb

```diff
@@ -77,6 +84,13 @@
           </div>
         <% end %>
       </div>
+
+    <% else %>
+      <div class="card-title mt-5 px-2">
+        <h2 class="name-bold">Tickets as a Teacher</h2>
+        <p>Looks like you're not teaching anyone yet.</p>
+        <%= link_to 'Add your available timeslots so people can book you', new_timeslot_path %>
```

Comment:
Do we need to pass the user here, or is this link working?

Reply:
link is working!